corehq/apps/users/tasks.py

```diff
@@ -36,6 +38,20 @@ def tag_docs_as_deleted(cls, docs, deletion_id):
     cls.get_db().bulk_save(docs)
 
 
+@task(rate_limit=2, queue='background_queue', ignore_result=True)  # 2 saves/sec for cloudant slowness
+def tag_forms_as_deleted_rebuild_associated_cases(formlist, deletion_id, deleted_cases=set()):
```

Comment:
Using a mutable type as the default value for an argument is generally a bad idea.  It's 'OK' here since you aren't changing the set at all but still not a good practice. Rather use `None` as the default value and then check for None in the function.